src/fft4g32.c

```diff
@@ -9,11 +9,12 @@
 
 #if WITH_CR32
 #include "rdft_t.h"
-static void * null(void) {return 0;}
-static void forward (int length, void * setup, double * H) {lsx_safe_rdft_f(length,  1, H); (void)setup;}
-static void backward(int length, void * setup, double * H) {lsx_safe_rdft_f(length, -1, H); (void)setup;}
+static void * null(int length) {return 0; (void)length;}
```

Comment:
I think it may cause unreachable code error on some compilers, although it's no-op.

How about this?
```suggestion
static void * null(int length) {(void)length; return 0;}
```

Reply:
Done